theme.js

```diff
@@ -41,12 +45,23 @@ define(function(require, exports, module) {
         // when plugin is loaded
         plugin.on("load", function() {
 
+            menuItem = new ui.item({
+                type: "check",
+                caption: "ToggleTheme",
+                onclick: toggleTheme
+            });
+
+            menus.addItemByPath(
+                "View/PresentationDiv", new ui.divider(), 3, plugin
+            );
+            menus.addItemByPath("View/Toggle Theme", menuItem, 4, plugin);
```

Comment:
Hey, this should be a checkable "Night Mode" above "Presentation Mode", per Asana.